### PR TITLE
Update links pointing to member permission on docs

### PR DESCRIPTION
**Pull request by JuanitoFatas**
Member permission was [renamed to Security last year](https://buildkite.com/changelog/206-new-security-settings-section). This PR updates a few missing links.

### CODE REVIEW

File: pages/apis/managing_api_tokens.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ Removing access from a token sends a notification email to the token's owner, wh
 
 ## Limiting API Access by IP address
 
-If you'd like to limit access to your organization by IP address, you can create an allowlist of IP addresses in the [organization's permission settings](https://buildkite.com/organizations/~/member-permissions).
+If you'd like to limit access to your organization by IP address, you can create an allowlist of IP addresses in the [API tab of organization's security settings](https://buildkite.com/organizations/~/security/api).
```
Comment (jameshill) [inline pages/apis/managing_api_tokens.md:65]: 👍🏻 looks good

File: pages/pipelines/notifications.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ You can set notifications:
 * On build status events in the Buildkite UI, by using your Slack Notification Service's 'Build State Filtering' settings.
 * On step status and other non-build events, by extending the Slack Notification Service using the `notify` attribute in your `pipeline.yml`.
 
-Before adding a `notify` attribute to your `pipeline.yml`, ensure an organization admin has set up a [Slack integration](/docs/integrations/slack) for the channel or user that you want to post to. Buildkite customers on the [Enterprise](https://buildkite.com/pricing) plan can also check the ['Manage Notifications Services'](https://buildkite.com/organizations/~/member-permissions) checkbox to create, edit, or delete notification services. For detailed information about setting up a Notification Service, see the [Slack integration page](/docs/integrations/slack).
```
Comment (jameshill) [inline pages/pipelines/notifications.md:116]: confirmed this is correct too
image

File: pages/apis/managing_api_tokens.md (hunk 1 above)
Comment (jameshill) [inline pages/apis/managing_api_tokens.md:65]:
```suggestion
If you'd like to limit access to your organization by IP address, you can create an allowlist of IP addresses in the [organization's API security settings](https://buildkite.com/organizations/~/security/api).
```

**Review by jameshill — COMMENTED** [review]
I've confirmed the URLs are correct.

And made a suggestion on the wording of link.

File: pages/apis/managing_api_tokens.md (hunk 1 above)
Comment (mbelton-buildkite) [reply pages/apis/managing_api_tokens.md:65]: +1

**Review by mbelton-buildkite — APPROVED** [review]
Looks good to me, with the suggestion from @jameshill 😊

**Comment by jameshill** [comment]
low how did we miss this from Hincy last year: https://github.com/buildkite/docs/pull/2149

**Pull request merged**